app/Http/Controllers/Two/UserController.php

```diff
@@ -135,7 +143,13 @@ public function show($id)
     {
         $user = User::findOrFail($id);
 
-        return $this->item($user);
+        $response = $this->item($user);
+
+        if (! Auth::user()) {
```

Comment:
Let's use `Auth::check` here instead of returning the full user model:

```suggestion
        if (! Auth::check()) {
```

Reply:
@DFurnes omg I didn't know that existed!!!

Reply:
Oh wait, or better yet `Auth::guest()`!